assets/search/components/TopicForm.tsx

```diff
@@ -134,6 +134,7 @@ const TopicForm: React.FC<IProps> = ({
                                 }
                             >
                                 <Dropdown
+                                    fullWidth={true}
```

Comment:
shouldn't this be actually handled already via `stretch` ?

Reply:
No the stretch is for the `button` only. Not the options popover.